### ISSUE TITLE
Don't have permissions to enumerate devices

### ISSUE THREAD

**Issue by mkarliner** [tijmenvangulik/cordova-usb-hid#3]
I'm using the first example you have given in a Vue3 application.
When I attempt to enumerate devices I get the following error.

`Uncaught (in promise) Can not enumerate devices: User has not given 10529/net.karliner.hidthingtest permission to access device /dev/bus/usb/001/002 `

At the moment, I can't see how to request usb enumeration permissions.
As you obviously have got your example running, could you tell me how you get them?

My target is Android 11 (moto9 plus)

Code as follows:

```
<script>
import { defineComponent } from "vue";

export default defineComponent({
  name: "IndexPage",
  async mounted() {
    console.log("Hello Thing");
    this.numhids = "3";

    console.log(window.cordova.plugins.UsbHid.registerReadCallback);
    await window.cordova.plugins.UsbHid.registerReadCallback((response)=>{
              console.log("received "+buf2hex(response));
    })
    var devices=await cordova.plugins.UsbHid.enumerateDevices();
    if (devices.length==0) throw "Device not found";
    await cordova.plugins.UsbHid.requestPermission(devices[0]);
    await cordova.plugins.UsbHid.open({
                                  packetSize:121,
                                  skippFirstByteZero:true,
                                  writeTimeout:500});
    var data= new Uint8Array([0x02,0xF0,0xFD,0x00,0x80,0x80,0xF2]);
    await cordova.plugins.UsbHid.write(data)
    console.log("Devidces ", devices)
  },
  data() {
    return {
      numhids: "0",
    };
  },
});
</script>
```

**Comment by tijmenvangulik** [comment]
At the time I wrote this it worked fine. You just had to do the things which I explained in the readme. Android must have changed the security requirements. Currently I am upgrading my own app to android 11. So I will run into the same problem. I will see what I can find. If you find anything I would like the hear it, maybe we can help each other.

**Comment by tijmenvangulik** [comment]
The repository was a combination of these repositories, maybe one of the has already fixed this

https://www.npmjs.com/package/cordova-plugin-usbserial

https://github.com/noconsulting/cordova-plugin-hid-usb/blob/master/src/android/fr/noconsulting/UsbHid.java

**Comment by mkarliner** [comment]
I've tried an enumeration test with this https://github.com/kyosho-/cordova-plugin-usb-event

and it worked.

**Comment by tijmenvangulik** [comment]
I have fixed this problem in version 1.0.4
